bin/baseplate-healthcheck

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!python
```

Comment:
`#!python` is a special shebang that wheel installers understand. They will rewrite this to the installation Python environment when the wheel is installed: https://packaging.python.org/en/latest/specifications/binary-distribution-format/#recommended-installer-features

This is actually what setuptools was doing already (it automatically rewrites these shebangs to `#!python` when it creates the wheel) which is why you don't see these shebangs in the wheel diff I posted. Poetry doesn't rewrite the shebangs automatically for these "legacy" scripts though, so I had to do it by hand.

Reply:
does this work in 3.8?

Reply:
Yes it does.